cfme/infrastructure/pxe.py

```diff
@@ -621,16 +625,19 @@ def get_pxe_server_from_config(pxe_config_name):
 
     pxe_config = conf.cfme_data.get('pxe_servers', {})[pxe_config_name]
 
-    return PXEServer(name=pxe_config['name'],
-                     depot_type=pxe_config['depot_type'],
-                     uri=pxe_config['uri'],
-                     userid=pxe_config.get('userid') or None,
-                     password=pxe_config.get('password') or None,
-                     access_url=pxe_config['access_url'],
-                     pxe_dir=pxe_config['pxe_dir'],
-                     windows_dir=pxe_config['windows_dir'],
-                     customize_dir=pxe_config['customize_dir'],
-                     menu_filename=pxe_config['menu_filename'])
+    return PXEServer(
+        parent=pytest.store.current_appliance.sentaku_app,
```

Comment:
Ahh I see this is to get it so that the Application is present and is no longer needed to be passed on object create right?